### DISCUSSION TITLE
Remove a listener

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#634] [Help] [answered]
I want to add/remove listeners, but I dont know how to do this. 
https://imgur.com/a/HPm3tPV
This is what I have tried, add listeners works, but the remove doesnt work.

**Comment by TheMode** [comment, marked as answer]
It is currently impossible to remove inline listeners. You need to call `addListener(EventListener)` and use the same listener object in `removeListener(EventListener)`.

**Reply by DevBlokJochem** [reply]
https://imgur.com/a/9IYolUk
This is what you mean right? (this works for me)

**Reply by TheMode** [reply]
Yes. Also you should include your code as text in GitHub directly.

**Reply by DevBlokJochem** [reply]
private val blockBreak = EventListener.of(PlayerBlockBreakEvent::class.java, CaptureBreakEvent::onBreak)

    private fun registerEvents() {
        captureNode.addListener(blockBreak)
    }

    private fun unRegisterEvents() {
        captureNode.removeListener(blockBreak)
    }

like this or?
